quickstart.sh

```diff
@@ -96,22 +96,46 @@ curl -s -X POST "http://localhost:9200/ecommerce-keyword/_aliases/ecommerce" -H
 echo -e "\n"
 
 echo -e "${MAJOR}Prepping Data for Ingestion\n${RESET}"
-if [ ! -f ./icecat-products-w_price-19k-20201127.tar.gz ]; then
+if [ ! -f ./esci.json.zst ]; then
   echo -e "${MINOR}Downloading the sample product data\n${RESET}"
-  wget http://querqy.org/datasets/icecat/icecat-products-w_price-19k-20201127.tar.gz
+  wget https://esci-s.s3.amazonaws.com/esci.json.zst
 fi
 
-if [ ! -f ./icecat-products-w_price-19k-20201127.json ]; then
+if [ ! -f ./esci.json ]; then
   echo -e "${MINOR}Unpacking the sample product data, please give it a few minutes!\n${RESET}"
-  tar xzf icecat-products-w_price-19k-20201127.tar.gz
+  zstd --decompress esci.json.zst 
+  # create a sample for starters
+  head -n 100000 esci.json > esci_100000.json
```

Comment:
The ESCI dataset is available under the Apache License so we could commit the JSON file instead of regenerating, assuming the size of a generated file is not extreme.

Reply:
In theory yes. We're a little cautious since we're not sure if the dataset can be licensed under Apache 2.0 in the first place. So we'd rather have the users download the data themselves (or switch to their own dataset) and not host a copy of the data.
With the latest changes the indexing is done within a few minutes, so it shouldn't be a huge issue for users.